src/porepy/fracs/fracture_network_3d.py

```diff
@@ -1036,8 +1036,16 @@ def _uniquify_points_and_edges(
         # fractures meeting in a line.
 
         # Do a sort of edges before looking for duplicates.
-        e_unique, unique_ind_e, all_2_unique_e = setmembership.unique_columns_tol(
-            np.sort(edges, axis=0)
+        # IMPLEMENTATION NOTE: The overall algorithm does not depend on preserving (or
+        # not preserving) the order of the edges, hence the simplest option would be to
+        # use the argument (which will preserve). However, altering the order may lead
+        # to different ordering of subdomains (fractures and their intersections), hence
+        # a rearrangement of the global degrees of freedom. Specifically, the test
+        # `TestMixedDimensionalUpwinding.test_3d_2d_1d_0d` relies on a specific ordering
+        # constructed without preserving the order (there are deep legacy reasons for
```

Comment:
Here I guess we can remove the implementation note? But I expected we had to change some test to make this work with np. If my memory is wrong, just ignore.

Reply:
Same here, tests did not have to change